libs/jsapi/nexus/LEAFNexusAPI.js

```diff
@@ -76,7 +76,7 @@ var NexusEmployeeAPI = function (baseAPIURL) {
         setCSRFToken = function (token) { csrfToken = token; },
 
         getByEmail = function (emailAddress, onSuccess, onFail) {
-            var fetchURL = apiURL + '/search&q=' + emailAddress + '&noLimit=0';
+            var fetchURL = apiURL + '/search?q=' + emailAddress + '&noLimit=0';
```

Comment:
I'm thinking these should be "let"s.  Here and the one(s) below.

Reply:
I have these vars updated.